topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.md

```diff
@@ -215,59 +238,84 @@ In order to convert the chromosome names we have therefore two things to do:
 
 
 ## Analysis
 
-Before comparing the 2 region files, we will add the promoter region to the gene records.
+Our goal is still to compare the 2 region files (the genes file and the peak file from the publication)
+to know which peaks are related to which genes. If you really only want to know which peaks are located **inside** genes you 
+can skip the next step. Otherwise, it might be reasonable to include the promoter region into the comparison, *e.g.* because 
+you want to include Transcriptions factors in ChIP-seq experiments.
 
 > ### {% icon hands_on %} Hands-on: Add promoter region to gene records
 >
-> 1. **Get Flanks** {% icon tool %}: Run **Get Flanks**, which returns flanking regions for every gene with the following settings:
->     - As **Select data** use the file from UCSC
->     - **Region**: `Around Start`
->     - **Location**: `Upstream`
->     - **Offset**: `10000`
->     - **Length**: `12000`
+> 1. **Get Flanks** {% icon tool %} with the following settings:
+>     - **Select data** to the file from UCSC
+>     - **Region** to `Around Start`
+>     - **Location** to `Upstream`
+>     - **Offset** to `10000`
+>     - **Length** to `12000`
+>
+>     This tool returns flanking regions for every gene
+>
+> 2. Compare the rows of the resulting BED file with the input to find out how the start and end positions changed
+>
+>    > ### {% icon tip %} Tip: Inspecting several files using the scratchbook
+>    >
+>    > * Click **Enable/Disable Scratchbook** on the top panel
+>    >
+>    >    ![Enable/Disable Scratchbook](../../images/intro_scratchbook_enable.png)
+>    > 
+>    > * Click on the **eye** icon of the files to inspect
+>    > * Click on **Show/Hide Scratchbook**
+>    >
+>    >    ![Show/Hide Scratchbook](../../images/intro_scratchbook_show_hide.png)
+>    {: .tip}
+>
+> 3. Rename your dataset to reflect your findings
 {: .hands_on}
 
-Now compare the resulting BED file with the input.
-Look at the contents and compare the rows in the input to the rows in the output to find out how the start and end positions changed.
-Rename the dataset (by clicking on the pencil icon) to reflect your findings.
-
-You might have noticed that the UCSC file is in `BED` format and has a database associated to it. That's what we want for our peak file as well:
+You might have noticed that the UCSC file is in `BED` format and has a database associated to it. That's what we want for our peak file as well
 
 > ### {% icon hands_on %} Hands-on: Change format and database
 >
 > 1. Click on the **pencil icon** in the history entry of our peak region file:
 >      ![Pencil icon](../../images/edit_icon.png)
 > 2. Switch to the `Convert Format` tab
 > 3. Select `Convert Genomic Intervals To BED` and press **Convert**
+> 4. Edit the "Database/Build" to select "mm9", the database build for mice used in the paper
```

Comment:
In my previous testing this was not necessary, which is why I removed a similar point in commit 29f1a970c87862c61b09c4c8170f09bd364adc25

Reply:
Not strictly needed, but there is a change that this helps people and in theory tools can check that the genome-build is the same ... e.g. intersect tools. We need a plan for the entire genome-build concept in Galaxy to go further. For the time being I would like to keep it and tell people that these thing exists.

Reply:
Maybe you could write something like: `Make sure the database built for mice used in the paper "mm9" is selected in "Database/Build"`

Reply:
I'm fine to change the sentence, but @nsoranzo point was to remove it I think. @nsoranzo how strong do you feel about it? I can remove it.

Reply:
I actually agree with @willdurand's suggestion, it's basically what I have used in https://github.com/galaxyproject/training-material/blob/master/topics/introduction/tutorials/galaxy-intro-101/tutorial.md#-icon-hands_on--hands-on-ucsc-genome-browser

Reply:
I added a similar sentence.

Reply:
Thanks!